MIP/mip-40/README.md

```diff
@@ -0,0 +1,83 @@
+
+### L2 Token Pool (Gas Pool)
+1. Burning gas an amount of gas $\alpha$ on the L2 MUST result in depositing that same amount of gas $\alpha$ into an L2 pool.
+2. The L2 pool MUST be recirculated to the L2 users in a manner that is not specified in this MIP. Generally, this means the L2 pool must either be used to issue rewards or otherwise be a reliable means of purchasing gas token on the L2. 
+
+A system adhering to these requirements is referred to as a Gas Pool. A more complete proposal for the usage of Gas Pools is provided in [MIP-n](todo).
+
+### Intermediary Tokens
+1. A complete implementation of *AB-FFS Decoupled* SHOULD use intermediary tokens to bridge the L1 and L2 systems. An implementation not using intermediary tokens is referred to as *AB-FFS Partially Decoupled*.
+2. The intermediary tokens MAY manage their own exchange rates, markets, and fees between each other, the L1 staking token, the L2 staking token, and the L1 LP token.
+
+A more complete proposal for the usage of intermediary tokens is provided in [MIP-n](todo).
+
+### Genesis
+[MD-38](https://github.com/movementlabsxyz/MIP/pull/38) requests provisions for genesis for any system addressing its desiderata. The genesis for *AB-FFS Decoupled* is not fully specified in this MIP and such full specification is provided under [MIP-n](todo). However, generally, there are two strategies for genesis:
+
+1. **Separate Mints**: mint the L1 staking token and the L2 gas token simultaneously in fixed supplies amounting to the intended total supply.
+2. **Genesis Bridge Transfer**: mint the L1 staking or L2 gas token in a fixed supply and transfer the other token from the L1 to the L2 s.t. a successful bridge transfer occurs before the system is operational.
+
+### On the Nature of the Tokens
+Fundamentally, the L1 staking and L2 gas token in an *AB-FFS Decoupled* system are independent, i.e., not constituting a representation of the same token redeemable on different chains. The comprise two closed token systems where assets cannot be lost or gained.
+
+The usage of *AB-FFS Decoupled* is then suitable for systems wherein the governing body wishes to prioritize the security of the total supply of the staking tokens. To the extent the governing body wishes for the economics of the L1 staking token and L2 gas token to remain correlated, the governing body shall make efforts to peg the value of the L1 LP token to the L1 staking token.
+
+In the case that *AB-FFS Decoupled* Grant is implemented, FFS shall require a new token generation event at some point determined by the size of the grant. Thus, FFS has a lifespan w.r.t. to the L1 staking token. This lifespan is further detailed in [MIP-n](todo).
+
+In the case that *AB-FFS Decoupled* Demurrage is implemented, the value of L1 LP Token shall tend to zero over time. This may also be remedies by new token generation events. This phenomenon is further detailed in [MIP-n](todo).
```

Comment:
```suggestion
In the case that *AB-FFS Decoupled Demurrage* is implemented, the value of L1 LP Token shall tend to zero over time. This may also be remedies by new token generation events. This phenomenon is further detailed in [MIP-n](todo).
```

Reply:
There are several battle-tested bridge designs. None of them have this issue of supply tending to zero. 
It is not because you "burn" tokens on the L2 that there is an issue. 
On the L2 you have an infinite supply of L2 tokens, what matters is that users holding(L2MOVE)  on L2 are equal to what is locked in the bridge contract on L1 (L1MOVE). 
This is the _lock-mint_ bridge design, the simplest to implement.

Reply:
>  None of them have this issue of supply tending to zero. It is not because you "burn" tokens on the L2 that there is an issue. On the L2 you have an infinite supply of L2 tokens...

Burning tokens on the L2 as gas without minting certainly means your will tend towards zero. We don't have an infinite supply unless we are allowed to make something else our gas token. 

> On the L2 you have an infinite supply of L2 tokens, what matters is that users holding(L2MOVE) on L2 are equal to what is locked in the bridge contract on L1 (L1MOVE).

I've explained why we can't guarantee this. 

> This is the lock-mint bridge design, the simplest to implement.

Yes, but we don't have an architecture which supports doing this while having a fixed supply, secure settlement, and sybil resistant gas.